app/controllers/invites_controller.rb

```diff
@@ -0,0 +1,33 @@
+class InvitesController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
+    @user = User.find params[:user_id]
+
+    @invites = @user.invites
+  end
+
+  def create
+    @event = Event.find params[:event_id]
+    @invitee = User.find params[:invitee_id]
+
+    @event.invite! @invitee
+
+    redirect_to :back
+  end
+
+  def destroy
+    @invite = Invite.find(params[:id])
+    @invite.destroy
+
+    redirect_to :back
+  end
+
+  def accept
+    @invite = Invite.find(params[:invite_id])
+
+    @invite.invitee.accept_invitation! @invite
```

Comment:
`@invite` twice in this line - might be a candidate for refactoring. Consider adding an instance method to `Invite` to do this work.